Resources/algorithm-results/project-files.php

```diff
@@ -1,22 +1,20 @@
 <?php
-$getProjectFilesText = function($isLiveMode) {
-    if ($isLiveMode) {
+    if ($pageName == "live") {
```

Comment:
Move variable definitions to the callers.  This page should be pure HTML and injections no logic